pkg/statistics/handle/storage/read.go

```diff
@@ -782,7 +794,14 @@ func loadNeededIndexHistograms(sctx sessionctx.Context, is infoschema.InfoSchema
 		tbl.LastAnalyzeVersion = max(tbl.LastAnalyzeVersion, idxHist.LastUpdateVersion)
 	}
 	tbl.SetIdx(idx.ID, idxHist)
-	statsHandle.UpdateStatsCache([]*statistics.Table{tbl}, nil)
+	tbl.LastAnalyzeVersion = max(tbl.LastAnalyzeVersion, idxHist.LastUpdateVersion)
```

Comment:
I think this line should be removed.

Reply:
Oh, it was accidentally introduced while I was resolving conflicts.

Reply:
Removed.